### ISSUE TITLE
training error 

### ISSUE THREAD

**Issue by ladyxuxu** [cowarder/Poly-YOLO#1]
hi，do you have met this error :RuntimeError: cannot perform reduction function max on tensor with no elements because the operation does not have an identity 

and Would you mind share me the version of pytorch you use ?

**Comment by cowarder** [comment]
You need to create a folder 'imgs' in 'simulator_dataset' to put simulator images mentioned in the paper. The newest stable pytorch GPU version is Ok.

**Comment by ladyxuxu** [comment]
yes , Mistakes occur during training , after  train iterations ,it random happen. could mind provide your e-mail  ,and  i attach the picture  for reference

**Comment by ladyxuxu** [comment]
![image](https://user-images.githubusercontent.com/32082799/91511768-0b369b80-e913-11ea-9fc9-0d86441aca0c.png)

**Comment by ladyxuxu** [comment]
![image](https://user-images.githubusercontent.com/32082799/91511976-826c2f80-e913-11ea-9c04-e50f19e6c48d.png)
i track , the  y_true[b, ..., 0:4] has value ,but   true_box = y_true[b, ..., 0:4][object_mask_bool[b, ..., 0]] always empty ,is 0 .

**Comment by cowarder** [comment]
This happens when the image doesn't exist, the dataset lacks some images. You can print the image path to check which of the images doesn't exist and delete it in the txt file.

**Comment by ladyxuxu** [comment]
![image](https://user-images.githubusercontent.com/32082799/91528817-c6bef600-e93a-11ea-80b2-74aa98d2320c.png)
as you say ,this is my test result ，img exist meanwhile  lableinformation exist ,so i dont understand ,what happened

**Comment by cowarder** [comment]
If you meet a randomly happening error, you should pay more attention to error samples. If you check the error image, you'll find there isn't any object(car) in the image.

**Comment by ladyxuxu** [comment]
yes , i open the picture and find no car in pic,  and  i want to know "img_r_805.png 0,189,1,202,0,1,201,0,202,0,189,1,192"  no car exist in pic ,What do these labels represent?

**Comment by cowarder** [comment]
Not sure, maybe just some dirty samples.
